specification.markdown

```diff
@@ -223,7 +223,7 @@ Within an EBML Schema the `<element>` uses the following attributes to define an
 | attribute name | required | definition |
 |----------------|----------|------------|
 | name           | Yes      | The official human-readable name of the EBML Element. The value of the name MUST be in the form of an NCName as defined by the [XML Schema specification](http://www.w3.org/TR/1999/REC-xml-names-19990114/#ns-decl). |
-| level          | Yes      | The level notes at what hierarchical depth the EBML Element may occur within an EBML Document. The Root Element of an EBML Document is at level 0 and the Elements that it may contain are at level 1. The level MUST be expressed as an integer; however, the integer may be followed by a '+' symbol to indicate that the EBML Element is valid at any higher level.  |
+| level          | Yes      | The level notes at what hierarchical depth the EBML Element may occur within an EBML Document. The Root Element of an EBML Document is at level 0 and the Elements that it may contain are at level 1. The level MUST be expressed as an integer. Note that for Elements defined as `global` and `recursive` the Element MAY occur at a level greater than or equal to the defined `level`.|
```

Comment:
That sentence seems to be a bit too complex for no particular reason. What about the slightly shorter "Note that Elements defined as `global` and `recursive` MAY occur…"? Then again I'm not a native speaker and may be off base here :)